cmd/kola/options.go

```diff
@@ -50,3 +57,20 @@ func init() {
 	sv(&kola.AWSOptions.InstanceType, "aws-type", "t1.micro", "AWS instance type")
 	sv(&kola.AWSOptions.SecurityGroup, "aws-sg", "kola", "AWS security group name")
 }
+
+// Sync up the command line options if there is dependency
+func syncOptions() error {
+	for k, v := range kolaDefaultImages {
```

Comment:
missed this before, no need to loop here since this is a map:

```go
if kola.QEMUOptions.DiskImage == "" {
    image, ok := kolaDefaultImages[kola.QEMUOptions.Board]
    if !ok {
        fmt.Errorf("unsupport board "%q", kola.QEMUOptions.Board)
    }
    kola.QEMUOptions.DiskImage = image
}